README.md

```diff
@@ -106,12 +106,13 @@ and [netservice](fab3/netservice.go)
 implementations. For details about limitations and implementations of the the
 instructions look at the [Fab3 Instructions](Fab3_Instructions.md).
 
-To create the Fab3 binary, this repository must be checked out in the [GOPATH](https://github.com/golang/go/wiki/GOPATH)
+To create the Fab3 binary, this repository must be checked out locally:
```

Comment:
This will work only for making the Fab3 binary. If the repo is checked out locally though, the integration tests will fail, and the tutorial will need to be updated/changed

Reply:
I figured there was still a need, but I was too lazy to try.